b2sdk/api.py

```diff
@@ -243,11 +250,15 @@ def create_bucket(
             replication=replication,
         )
         bucket = self.BUCKET_FACTORY_CLASS.from_api_bucket_dict(self, response)
-        assert name == bucket.name, 'API created a bucket with different name\
-                                     than requested: %s != %s' % (name, bucket.name)
-        assert bucket_type == bucket.type_, 'API created a bucket with different type\
-                                             than requested: %s != %s' % (
-            bucket_type, bucket.type_
+        assert name == bucket.name, (
+            'API created a bucket with different name\
+                                     than requested: %s != %s'
+            % (name, bucket.name)
+        )
+        assert bucket_type == bucket.type_, (
+            'API created a bucket with different type\
+                                             than requested: %s != %s'
+            % (bucket_type, bucket.type_)
```

Comment:
No! God please no!

Reply:
🤮

Most of our issues seem to be caused by line the width limiter 🤔

Reply:
Splitting the string into two lines with `\` was not done by black here and it wouldn't make much sense to try to manually align things like that if you're using a formatter like black. Though to be fair, even after fixing that the formatting is a little awkward, black can sometimes try to split on `%`. Maybe use f-strings? :D